alembic/versions/2ee259ae22e5_add_organizer_column_event.py

```diff
@@ -0,0 +1,25 @@
+"""add-organizer-column(event)
+
+Revision ID: 2ee259ae22e5
+Revises: 98e4dbfc868a
+Create Date: 2019-11-06 11:02:22.545841
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '2ee259ae22e5'
+down_revision = '98e4dbfc868a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('events', sa.Column(
+        'organizer_email', sa.String(), nullable=True))
```

Comment:
I just checked Samuel's PR and I saw that the organizer email is sent as a list since it is taking two emails, is this data type appropriate for storing it?

Reply:
I just wanna know from the list of 2 emails, how will I know which one is the actual organizer_email @mifeille